### PR TITLE
Admin create categories

### PR DESCRIPTION
**Pull request by HUSTToanLH**

### CODE REVIEW

File: app/views/admin/categories/_left.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      </ul>
+    </li>
+    <li>
+      <div class="col-md-12 textbox text-justify">
+        <%= category.description %>
+      </div>
+    </li>
+    <li>
+      <div class="btn btn-default col-md-3 col-md-offset-9">
+        <%= link_to t "views.categories.start" %>
```
Comment (vinhnguyen-fly) [inline app/views/admin/categories/_left.html.erb]: link to where ?

File: app/views/admin/categories/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <%= form_for [:admin, @category] do |f| %>
+      <%= f.label :name, t("admin.category.name") %>
+      <%= f.text_field :name, class: "form-control" %>
+
+      <%= f.label :description, t("admin.category.description") %>
+      <%= f.text_area :description, class: "form-control" %>
+
+      <%= f.submit t("admin.category.create"), class: "btn btn-default col-md-3" %>
+    <% end %>
+
```
Comment (vinhnguyen-fly) [inline app/views/admin/categories/new.html.erb]: remove this line

File: config/routes.rb [hunk 3]
```diff
@@ -7,10 +7,11 @@
...
   end
   resources :sessions, only: [:new, :create, :destroy]
-  resources :categories, only: :index
+  resources :categories, only: [:index, :new, :create]
   resources :relationships, only: [:index, :create, :destroy]
   namespace :admin do
-    root "users#index"
+    root "categories#index"
     resources :users
+    resources :categories
```
Comment (vinhnguyen-fly) [inline config/routes.rb]: if you don't use all method, you must register which method you use

File: db/migrate/20160322070745_create_categories.rb [hunk 4]
```diff
@@ -5,5 +5,6 @@ def change
 
       t.timestamps null: false
     end
+    add_column :categories, :description, :text
```
Comment (vinhnguyen-fly) [inline db/migrate/20160322070745_create_categories.rb]: move in side block create table:
```
t.text :description
```

File: app/controllers/admin/categories_controller.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+class Admin::CategoriesController <ApplicationController
+  before_action :logged_in_user
+  before_action :verify_admin
+  before_action :find_category, except: [:index, :new, :create]
```
Comment (vietmt) [inline app/controllers/admin/categories_controller.rb]: except: [:index, :new, :create] -> điều kiện như thế thì khi nào chạy action này?

**Pull request merged**